### ISSUE TITLE
Cuckoomon should click open on IE file download prompt

### ISSUE THREAD

**Issue by seanthegeek** [ctxis/CAPE#315]
When submitting a URL to a file with the ie package, cuckoomon should click on the open button to open the sample, so it can sun.

It looks like code for this already exists, but does not work?

https://github.com/ctxis/CAPE/blob/master/analyzer/windows/modules/auxiliary/human.py#L48

**Comment by kevoreilly** [comment]
Do you see a line like this in your analysis logs?

`2019-03-15 14:40:57,744 [root] DEBUG: Started auxiliary module Human`

**Comment by seanthegeek** [comment]
Yep. And the mouse is moving as expected.

**Comment by kevoreilly** [comment]
Hmm I'm not sure I properly read the issue here - sorry about that. I misread this as an issue with the 'ie' package, or with the human.py auxiliary module. But actually the ie package isn't intended for downloading and executing samples directly from a URL, rather it's meant for malicious sites that might exploit the browser in a drive-by-download style.

In order to download and execute directly from a URL I actually made a start on a solution to this - it's called the 'wget' package and I made it in response to issue #126 which is essentially the same issue.

Currently there is a limitation with the wget package in that it doesn't perform static analysis on the downloaded file - a bit more work is needed to allow for a job to be submitted without a sample, and the sample to be retrospectively 'inserted' in all the aspects of the analysis chain. Still, it does the basic job of downloading and executing, capturing the behaviour so give that a try and let me know how you get on.
